sal_interfaces/ESS/ESS_Telemetry.xml

```diff
@@ -292,4 +292,82 @@
       <Count>1</Count>
     </item>
   </SALTelemetry>
+  <SALTelemetry>
+    <EFDB_Topic>ESS_hx85a</EFDB_Topic>
+    <Description>Omega HX85A Humidity Sensors</Description>
```

Comment:
These measure humidity and temperature. Also I suggest singular because this item is one reading from one sensor. Please consider "Humidity Sensors" ->  "Humidity and Temperature Sensor" or similar.

Reply:
I changed it to `Omega HX85A Humidity, Temperature and Dew Point Sensor`.